packages/medusa/src/api/admin/returns/[id]/request-items/route.ts

```diff
@@ -0,0 +1,40 @@
+import { requestItemReturnWorkflow } from "@medusajs/core-flows"
+import {
+  ContainerRegistrationKeys,
+  remoteQueryObjectFromString,
+} from "@medusajs/utils"
+import {
+  AuthenticatedMedusaRequest,
+  MedusaResponse,
+} from "../../../../../types/routing"
+import { AdminPostReturnsRequestItemsReqSchemaType } from "../../validators"
+
+export const POST = async (
+  req: AuthenticatedMedusaRequest<AdminPostReturnsRequestItemsReqSchemaType>,
+  res: MedusaResponse
+) => {
+  const { id } = req.params
+
+  const remoteQuery = req.scope.resolve(ContainerRegistrationKeys.REMOTE_QUERY)
+
+  await requestItemReturnWorkflow(req.scope).run({
```

Comment:
I beleive we should return the `result` of this workflow, which will be the preview of the Order after the change actions are applied.

Reply:
Do you think we will need anything but the totals preview in the FE? 

Was just discussing the response with Seb, and we thought of doing:
```
res.status(200).json({ return: orderReturn, total_summary: result.summary })
```

...provided the `result` here is the order preview.